cognitive/src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -28,11 +28,12 @@ class DetectedLanguageSuiteV4 extends TestBase with DataFrameEquality with TextK
   def getDetector: LanguageDetectionV4 = new LanguageDetectionV4()
     .setSubscriptionKey(textKey)
     .setLocation("eastus")
+    .setOptions(TextAnalyticsRequestOptionsV4("", true, false))
```

Comment:
what are the 3 params here?

Reply:
model version , stats, disable service log